code/modules/mob/living/basic/heretic/star_gazer.dm

```diff
@@ -0,0 +1,114 @@
+
+/mob/living/basic/star_gazer
+	name = "Star Gazer"
+	desc = "A creature that has been tasked to watch over the stars."
+	icon = 'icons/mob/nonhuman-player/96x96eldritch_mobs.dmi'
+	icon_state = "star_gazer"
+	icon_living = "star_gazer"
+	pixel_x = -32
+	base_pixel_x = -32
+	basic_mob_flags = DEL_ON_DEATH
+	mob_biotypes = MOB_HUMANOID
+	faction = list(FACTION_HERETIC)
+	response_help_continuous = "passes through"
+	response_help_simple = "pass through"
+	speed = -0.2
+	maxHealth = 3000
+	health = 3000
+
+	obj_damage = 400
+	melee_damage_lower = 25
+	melee_damage_upper = 25
+	combat_mode = TRUE
+	sentience_type = SENTIENCE_BOSS
+	attack_verb_continuous = "ravages"
+	attack_verb_simple = "ravage"
+	attack_vis_effect = ATTACK_EFFECT_SLASH
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	speak_emote = list("growls")
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	death_sound = 'sound/magic/cosmic_expansion.ogg'
+
+	unsuitable_atmos_damage = 0
+	unsuitable_cold_damage = 0
+	unsuitable_heat_damage = 0
+
+	gold_core_spawnable = HOSTILE_SPAWN
```

Comment:
More review incoming after I have my dinner but you absolutely do NOT want this to be spawnable from gold slime cores.
```suggestion
	gold_core_spawnable = NO_SPAWN
```

Reply:
whoops forgot